src/fish_indent.cpp

```diff
@@ -254,9 +254,8 @@ void prettifier_t::prettify_node(const parse_node_tree_t &tree, node_offset_t no
                 }
                 wcstring unescaped{source, node.source_start, node.source_length};
                 // Unescape the string - this leaves special markers around if there are any expansions or anything.
-                // TODO: This also already computes backslash-escapes like \u or \x.
-                // We probably don't want that - if someone picked `\x20` to spell space, they have a reason.
-                unescape_string_in_place(&unescaped, UNESCAPE_SPECIAL);
+                // We specifically tell it to not compute backslass-escapes like \U or \x, because we want to leave them intact.
```

Comment:
`s/ss/sh/`

Reply:
Yeah, a backlass is something entirely different.